chapters/2-processing/3-thresholding/thresholding.md

```diff
@@ -1092,7 +1092,7 @@ im = im / im.max()
 im = (im * 255).astype(np.uint8)
 
 # Apply triangle threshold
```

Comment:
Looks like I got the comment wrong too :) Could you add it to the same PR please, so that just one rebuild of the entire site is triggered?

Reply:
```suggestion
# Apply local Niblack threshold
```